pkg/rpc/context.go

```diff
@@ -82,7 +82,7 @@ var SourceAddr = func() net.Addr {
 	return nil
 }()
 
-var enableRPCCompression = envutil.EnvOrDefaultBool("COCKROACH_ENABLE_RPC_COMPRESSION", false)
+var enableRPCCompression = envutil.EnvOrDefaultBool("COCKROACH_ENABLE_RPC_COMPRESSION", true)
```

Comment:
Ah, I was imagining a change to the setting would affect new connections. I forgot that we would want to enable it on responses from the server as well. Yeah, let's leave this as an env var.